website/blog/2021-09-01-angular-10-update.md

```diff
@@ -25,18 +25,26 @@ Please note that this Angular update does not include Ivy Engine but covers some
 
 ## How WaveMaker can Help
 
```

Comment:
//Better to consider these... rewrite as needed

This impact building WaveMaker apps outside WaveMaker using your own CI/CD system.
So we expect you to upgrade node and npm on your build infrastructure with above mentioned versions.
Or you can utilize resources given by WaveMaker Team.
WaveMaker team is publishing Dockerfile, Docker image for this.

- Docker file.
  - Have commands to install required software utilities for building WaveMaker inside the container. 
  - Instructions to Build docker image out of this file and use docker image to build WaveMaker App.
  - Link to Docker Github and Dockerhub
- Docker Image
  - PreBaked Docker Image by WaveMaker team.
  - Customers utilize this to build a WaveMaker app by passing source code as  volume while running containers out of this image.
  - Docker Image URL on Docker hub
- Docker Image with NPM and Maven dependencies.
- PreBaked Docker Image by WaveMaker team.
- Same as point:2, but with Maven and NPM dependencies are pre baked.
- So Customers can utilize this image to build WaveMaker apps without contacting Maven or NPM repositories. 
- Docker hub url